BUILD.md

```diff
@@ -23,7 +23,11 @@ Use the provided CMakeLists.txt with [CMake](https://cmake.org) to generate a bu
 
 Building on Android is done using the [Gradle Build Tool](https://gradle.org/):
 
+Set Android SDK/NDK by environment variable `ANDROID_SDK_ROOT`/`ANDROID_NDK_HOME`.
+
 ```
+export ANDROID_SDK_ROOT=<path-to-SDK>
```

Comment:
This only applies to Linux I guess. Can you take this oustide of the ode block and add a note that this is only required on that platform?

Reply:
Thanks for your reminding. The user should know the way how to set environment variables on different platform. Delete that export command.